api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -314,6 +315,12 @@ message Watchdog {
   // duration assume a true deadlock and kill the entire Envoy process. Set to 0
   // to disable this behavior. If not specified the default is 0 (disabled).
   google.protobuf.Duration multikill_timeout = 4;
+
+  // Defines the maximum skew used to adjust the kill_timeout if kill_timeout is
```

Comment:
Two nits: 
1. Can you move this next to `kill_timeout` for readability? It's fine to have discontinuous numbering.
2. For field names, either write `*kill_timeout*` if the field is nearby, or use a `:ref:` links (grep for `"ref:.*envoy_api_field"` for examples.

Reply:
Done. Thanks for the pointers.